.prettierrc.json

```
@@ -1,4 +1,4 @@
 {
-  "semi": false,
-  "endOfLine": "lf"
+  "plugins": ["@prettier/plugin-php"],
+  "parser": "php"
```

If we want to use prettier as the PHP formatter, then we may need two configs, one for PHP and one for everything else? Or can we use a single config.
```suggestion
  "plugins": ["@prettier/plugin-php"],
  "parser": "php",
  "plugins": ["@prettier/plugin-php"],
  "semi": false,
  "singleQuote": true,
  "jsxSingleQuote": true,
  "endOfLine": "lf"
```